tools/sklearn/feature_selection.xml

```diff
@@ -103,11 +103,14 @@ res.to_csv(path_or_buf="$outfile", sep='\t', index=False)
         <expand macro="feature_selection_all">
             <expand macro="fs_selectfrommodel_prefitted"/>
         </expand>
-        <expand macro="feature_selection_output_mothods" />
+        <param name="save" type="boolean" truevalue="booltrue" falsevalue="boolflase" checked="false" label="Save the fitted selector?"/>
```

Comment:
Again a typo: `falsevalue="boolflase"`. It should be `falsevalue="boolfalse"` :)